contrib/babelfishpg_tsql/sql/import_export_compatibility.sql

```diff
@@ -59,9 +59,15 @@ CREATE OR REPLACE FUNCTION is_member(IN a VARCHAR) RETURNS INT
 AS 'babelfishpg_tsql', 'is_member' LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
 GRANT EXECUTE ON FUNCTION is_member(IN VARCHAR) TO PUBLIC;
 
-CREATE OR REPLACE FUNCTION schema_id(IN schema_name VARCHAR) RETURNS INT
-AS 'babelfishpg_tsql', 'schema_id' LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
-GRANT EXECUTE ON FUNCTION schema_id(IN VARCHAR) TO PUBLIC;
+-- Two declarations of schema_id based on number of parameters.
+--However, both call the same C function
+CREATE OR REPLACE FUNCTION schema_id()
+RETURNS INT AS 'babelfishpg_tsql', 'schema_id' LANGUAGE C IMMUTABLE PARALLEL SAFE;
+GRANT EXECUTE ON FUNCTION sys.schema_id() TO PUBLIC;
```

Comment:
This should not be immutable, lets make it stable since immutable can cache the schema id as a literal in many cases and later if there are drop schemas then we will return schema_id for a schema that does not exist

Reply:
Made the change in latest commit, Thanks.